woocommerce/woocommerce-ios
# Present Fancy alert informing of store switcher and name of store in My Store tab

Fixes #679 

- Set the name of the store as title of the view controller presented in _My store_ tab. The tab item remains unchanged.
- Present a fancy alert on app launch informing users that can switch stores. The alert is presented only once.

This PR is marked as *Not Ready to Merge* as the illustration in the fancy alert has not been added yet.

The store name is in the dashboard title. The tab bar item is not changed:
![simulator screen shot - iphone x - 2019-02-15 at 13 22 49](https://user-images.githubusercontent.com/2722505/52836225-db56e700-3124-11e9-8a17-692cf27a4526.png)

This is the fancy alert (illustration pending):
![simulator screen shot - iphone x - 2019-02-15 at 13 22 46](https://user-images.githubusercontent.com/2722505/52836246-f1fd3e00-3124-11e9-95b4-56741218d159.png)

And tapping the `Try It Now` button navigates to Settings:
![screen recording 2019-02-15 at 1 22 13 pm mov](https://user-images.githubusercontent.com/2722505/52836299-24a73680-3125-11e9-8ad3-7ed23cf7a6a5.gif)

## Testing
- Check out the branch
- To test the alert, edit the `shouldPresentStoreSwitcher` method in `SwitchStoreAlertLauncher` to always return `true`
- Build and run.
